hr_employee_health/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'HR Employee Health',
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr',
+    'author':
+        'Braibean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Allows storing information about employee\'s health',
+    'depends': [
+        'hr',
+    ],
+    'data': [
```

Comment:
@alexey-pelykh IMO manage sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
fixed